schemas/multidim-schema.json

```diff
@@ -108,13 +108,19 @@
                                     {
                                         "type": "string"
                                     },
+                                    {
+                                        "type": "number"
```

Comment:
I noticed we use IDs in some existing configs, and it is supported, so I added the option here.

Reply:
OK, I removed that change.

Reply:
@pabloarosado just to confirm this